static/elements/chromedash-feature-table.js

```diff
@@ -22,6 +24,8 @@ class ChromedashFeatureTable extends LitElement {
     this.starredFeatures = new Set();
```

Comment:
`starredFeatures` is being passed from the parent DOM. Does it still need to be intialized?

Reply:
IIRc, this is the recommended way to initialize properties as per the lit-element docs.  In this case, the value is immediately set again based on an attribute set in the DOM, however web components are meant to be reused in a variety of contexts so it might not always be set that way.